azext_edge/edge/providers/support_bundle.py

```diff
@@ -94,6 +103,8 @@ def build_bundle(
                     bundle = bundle_method(deployed_apis)
                 elif service_moniker == OpsServiceType.mq.value:
                     bundle = bundle_method(deployed_apis, log_age_seconds, include_mq_traces)
+                elif service_moniker == OpsServiceType.billing.value:
+                    bundle = bundle_method(deployed_apis, log_age_seconds, child_group=BILLING_RESOURCE_KIND)
```

Comment:
As indicated in the related commend, build_bundle doesn't need to know about support bundle hierarchy / we should remove reference to BILLING_RESOURCE_KIND.